### ISSUE TITLE
AIND T2 Keras Lesson 3.32 mismatch of loss function

### ISSUE THREAD

**Issue by jg1141** [udacity/aind-issue-reports#178]

**Comment by jg1141** [comment]
On [this page](https://classroom.udacity.com/nanodegrees/nd889/parts/16cf5df5-73f0-4afa-93a9-de5974257236/modules/6124bd95-dec2-44f9-bf3b-498ea57699c7/lessons/47f6c25c-7749-4a02-b807-7a5b37f362e8/concepts/7c3d9b4c-206d-4430-9459-c78357181d9a) the TEXT says "We'll specify the loss function to be `binary_crossentropy`  ..." but the CODE snippet immediately below has a loss function of `categorical_crossentropy`:
```
model.compile(loss="categorical_crossentropy", 
```

**Comment by letyrodridc** [comment]
Hi @jg1141 It's fixed.
![fixed](https://user-images.githubusercontent.com/9123307/33642111-2a88fbe4-da17-11e7-90c2-c859f3e7fc12.png)
I will close the issue.
Thanks so much for reporting this.

**Comment by cgearhart** [comment]
Previously fixed
